src/PKSim.Infrastructure/Serialization/Xml/XmlSerializationManager.cs

```diff
@@ -194,8 +196,12 @@ private void updatePropertiesFor<TObject>(TObject deserializedObject, int versio
          if (lazyLoadable != null)
             lazyLoadable.IsLoaded = true;
 
+         var buildingBlock = deserializedObject as IPKSimBuildingBlock;
+         if (buildingBlock != null)
+            buildingBlock.HasChanged = conversionHappened;
```

Comment:
if converted and building block, it will also be marked as changed so that it will be serialized